openapi.yml

```diff
@@ -339,19 +339,6 @@ components:
       required:
         - identifier
         - type
-    Administrative:
```

Comment:
Changes here look funky because I fixed the alpha sort in cocina-models openapi.yml. We're feeling the pain with this bump so that we never have to feel it again.